charts/sonarqube/README.md

```diff
@@ -255,7 +255,7 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `annotations`           | SonarQube Pod annotations                                                                                                               | `{}`               |
 | `edition`               | SonarQube Edition to use (e.g. `community`, `developer` or `enterprise`). Please note that the default `community` value is deprecated. | `community`        |
 | `community.enabled`     | Install SonarQube Community Build. When set to `true`, this parameter replaces `edition=community`                                      | `true`             |
-| `community.buildNumber` | The SonarQube Community Build number to install                                                                                         | `10.7.0`           |
+| `community.buildNumber` | The SonarQube Community Build number to install                                                                                         | `10.8.0`           |
```

Comment:
community.buildNumber should not be 10.8.0 ?

Reply:
Nice catch!